<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow1TRIALS</class>
 <widget class="QMainWindow" name="MainWindow1TRIALS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QCHECKBOX</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>200</y>
      <width>77</width>
      <height>76</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="chkbx_1">
       <property name="text">
        <string>Option 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkbx_2">
       <property name="text">
        <string>Option 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkbx_3">
       <property name="text">
        <string>Option 3</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
